CHANGELOG.rst

```diff
@@ -19,6 +19,8 @@ Changed
 * The BlueZ D-Bus backend implements a services cache between connections to significancy improve reconnect performance.
   To use the cache, call ``connect`` and ``get_services`` with the ``dangerous_use_bleak_cache``
   argument to avoid services being resolved again.
+* ``AdvertisementData`` class now has an attribute ``tx_power``. scanner.py in backends of winrt, corebluetooth and 
+  bluezdbus were changed to set tx_power if available.
```

Comment:
Can we get this for the Android backend too (even if you aren't able to test it)?

i.e. use `result.getTxPower()` in `_PythonScanCallback.onScanResult()`.

It also looks like we might want to convert `ScanResult.TX_POWER_NOT_PRESENT` to `None` to be consistent with other backends.

https://developer.android.com/reference/android/bluetooth/le/ScanResult#getTxPower()

Reply:
Yes. Actually now I have Android Studio but don't have test environment Android (will come in 2 weeks). So this time I made changes and test them later (if necessary). Thank you for your advice.